clients/rust/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(deprecated)]
```

Comment:
This is a bit annoying, but the generated clients implement deprecated traits, so this seemed like the easiest way to bump everything.